src/jsparse.c

```diff
@@ -2338,12 +2341,13 @@ NO_INLINE void jspeBlock() {
   return;
 }
 
-NO_INLINE JsVar *jspeBlockOrStatement() {
+NO_INLINE JsVar *jspeBlockOrStatement(bool *goAgain) {
   if (lex->tk=='{') {
     jspeBlock();
+    *goAgain = true;
```

Comment:
This looks a bit dodgy to me? Could we end up with a case where `goAgain` is null? We do call `jspeStatement(0);`

Reply:
Yes very much so - there's a bunch of calls where arguments are missing too, got part way through then thought I'd best check the approach